### PR TITLE
improved docstyle in _utils.py; SeqRecord.py and Bio/phenotype/ (PEP257)

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses issue #1473

### CODE REVIEW

File: Bio/_utils.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def iterlen(items):
 
 
 def read_forward(handle):
-    """Reads through whitespaces, returns the first non-whitespace line."""
+    """Read through whitespaces, returns the first non-whitespace line."""
```
Comment (peterjc) [inline Bio/_utils.py]: Should probably be "Read through whitespaces, return the first non-whitespace line."

**Comment by peterjc** [comment]
Looks good - can you add yourself to the ``NEWS.rst`` file for the upcoming release please?

**Comment by Gasta88** [comment]
Looks like I'm already there.

**Comment by peterjc** [comment]
OK, perhaps my local copy was out of date, merging - thank you!

**Pull request merged**